packages/features/bookings/lib/handleNewBooking.ts

```diff
@@ -979,10 +979,45 @@ async function handler(
 
   const allCredentials = await getAllCredentials(organizerUser, eventType);
 
-  const isOrganizerRescheduling = organizerUser.id === userId;
```

Comment:
Renamed `isOrganizerRescheduling` to `isBeingBookedByOrganizer` because it isn't just possibly true for rescheduling. It can also be true for fresh booking.